insights/parsers/__init__.py

```diff
@@ -329,6 +329,17 @@ def keyword_search(rows, **kwargs):
     Takes a list of dictionaries and finds all the dictionaries where the
     keys and values match those found in the keyword arguments.
 
+    Keys in the row data have ' ' and '-' replaced with '_', so they can
+    match the keyword argument parsing.  For example, the keyword argument
+    'fix_up_path' will match a key named 'fix-up path'.
+
+    In addition, several suffixes can be added to the key name to do partial
+    matching of values:
+
+    * '__contains' will test whether the data value contains the given
+      value.
+    * '__startswith' tests if the data value starts with the given value
+
```

Comment:
@PaulWay do you think it would be useful to include examples using these suffixes in the *Examples* section?